lib/util/traverser.js

```diff
@@ -40,6 +156,27 @@ class Traverser extends estraverse.Controller {
     static getKeys(node) {
         return Object.keys(node).filter(key => !KEY_BLACKLIST.has(key));
     }
+
+    /**
+     * Traverse the given AST tree.
+     * @param {ASTNode} node The root node to traverse.
+     * @param {Object} options The option object.
+     * @param {Object} [options.keys=DEFAULT_VISITOR_KEYS] The keys of each node types to traverse child nodes. Default is `./default-visitor-keys.json`.
+     * @param {Function} [options.enter=noop] The callback function which is called on entering each node.
+     * @param {Function} [options.leave=noop] The callback function which is called on leaving each node.
+     * @returns {void}
+     */
+    static traverse(node, options) {
+        new Traverser().traverse(node, options);
+    }
+
+    /**
+     * The default visitor keys.
+     * @type {Object}
+     */
+    static get DEFAULT_VISITOR_KEYS() {
+        return DEFAULT_VISITOR_KEYS;
+    }
 }
 
 module.exports = Traverser;
```

Comment:
Are consumers of this module expected to construct instances of `Traverser`, or are they only expected to use `Traverser.traverse`? If they should only use `Traverser.traverse`, then we could reduce the API surface by only exporting `traverse`:

```diff
-module.exports = Traverser;
+module.exports = Traverser.traverse;
```